src/main/java/com/hillel/vynnyk/homeworks/projectPersonSpring/service/AccountService.java

```diff
@@ -0,0 +1,57 @@
+package com.hillel.vynnyk.homeworks.projectPersonSpring.service;
+
+import com.hillel.vynnyk.homeworks.projectPersonSpring.entity.Account;
+import com.hillel.vynnyk.homeworks.projectPersonSpring.repo.AccountRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class AccountService {
+    private final AccountRepository accountRepository;
+
+    public List<AccountDto> findAccounts() {
+        return accountRepository.findAll().stream()
+                .map(this::mapAccount)
+                .toList();
+    }
+
+    public AccountDto findAccount(String id) {
+        return accountRepository.findByUid(id)
+                .map(this::mapAccount)
+                .orElseThrow();
+    }
+
+    public void deleteAccount(String uid) {
+        var account = accountRepository.findByUid(uid).orElseThrow();
+        accountRepository.delete(account);
+    }
+
+    public AccountDto updateAccount(String uid, AccountDto accountDto) {
+        var accountToUpdate = accountRepository.findByUid(uid).orElseThrow();
+        accountToUpdate.setIban(accountDto.iban());
+        return mapAccount(accountRepository.save(accountToUpdate));
+    }
+
+    private AccountDto mapAccount(Account account) {
+        return AccountDto.builder()
+                .id(account.getUid())
+                .iban(account.getIban())
+                .balance(account.getBalance())
+                .person_id(account.getPerson_id())
```

Comment:
```suggestion
                .personId(account.getPersonId())
```
все ж таки це стиль Джави.